### ISSUE TITLE
dist broken, process.env !== 'production' instead of process.env.NODE_ENV !== 'production'

### ISSUE THREAD

**Issue by CanRau** [system-ui/theme-ui#423]
Title basically says it all, not sure what happens, I installed latest theme-ui 0.2.44 and all index files in /dist are missing the NODE_ENV part from this line [packages/theme-ui/src/provider.js#L87](https://github.com/system-ui/theme-ui/blob/master/packages/theme-ui/src/provider.js#L87)

so it looks like
```js
if (process.env !== 'production') {
    if (outer.emotionVersion !== package_json.version) {
```

I'm using it with typescript, webpack and electron

Fixing the line manually in node_modules works fine

Update: in another electron project, using babel instead of typescript is works despite the fact that the lines look the same and `NODE_ENV` is missing as well

**Comment by CanRau** [comment]
Awesome 🎉 Thanks for this amazing project by the way love where it's heading 👍

**Comment by jxnblk** [comment]
Thanks! Should be fixed in `v0.2.46`, but please reopen this if you're still seeing issues
